static/locales/en-US.yaml

```
@@ -141,6 +141,7 @@ User Playlists:
   You have no playlists. Click on the create new playlist button to create a new one.: You have no playlists. Click on the create new playlist button to create a new one.
   Empty Search Message: There are no videos in this playlist that matches your search
   Search bar placeholder: Search in Playlist
+  Search in Videos: Search in Videos
```

This could be more specific imo

```suggestion
  Search in Videos: Search for Videos in Playlist
```

---

This is not searching for videos, but searching for playlist(s) with matching videos
No idea how to make it short